### ISSUE TITLE
open/close all option

### ISSUE THREAD

**Issue by sondra-kinsey** [warren-bank/moz-json-data-view#8]
This is a feature request.
I would really like it if there was an option to open or close ALL elements in the tree. By default, it seems to leave all elements closed. I would like two options in the context menu for any object in the tree to open all and close all descendents. The context menu for clicking in empty space in a JSON document (not an object) could have similar options to open or close all objects in the tree.
That is my very specific proposal, but my greatest need is to expand all elements easily, without clicking each object individually.

**Comment by warren-bank** [comment]
hi there..

first off, I hope you like the add-on :grinning: 

in response to your feature request..
have you tried changing the default [preference value](https://github.com/warren-bank/moz-json-data-view#user-content-user-preferences): `expand all nodes` ..to `true`?

I assume that'll accomplish what it is that you're asking for.
Well.. there are no context menus and such;
but this preference value is applies to all json pages when they load.

**Comment by warren-bank** [comment]
well.. though I've always open to ideas to improve usability, it's been a week and no response.. so I'm going to close this ticket.

nevertheless, feel free to post additional feedback.. I'll see it.

thanks :grinning:

**Comment by warren-bank** [comment]
hi again..

just wanted to let you know that I had a little time,
so I added a few tweaks to this project onto my "to-do" list.

the first one was regarding your request for an improved way to "expand all".

I just released an update, tagged: [v1.13|https://github.com/warren-bank/moz-json-data-view/releases/tag/v1.13], which adds the ability to recursively expand or collapse any branch using: Ctrl + click

Sorry about not doing the whole context-menu thing.. I'm just not a very big fan of unnecessary UI.
I hope this satisfies your needs.
